idl/dsthssp_calib.pro

```diff
@@ -532,8 +531,7 @@ case uvalue of
 ;TEMPOLARY 2016.12.8 8542A
 ;bad=[116,104,12,29,40,60,100,120,150,160,190,220,240,250,260,320,350,377,378,379,where(hd.polstate eq 45)]
 ;TEMPOLARY 2016.12.8 10830A
-
-bad=[105,104,135,324,343,344,323,322,26,11,where(hd.polstate eq 45)]
```

Comment:
bad is now unnecessary since trigger issue solved.

Reply:
We don't need to specify "Bad" in this program. If there are cloudy dataset or anything else, just set the corresponding weight to 0.                                --- T.Anan